Makefile

```diff
@@ -32,6 +33,7 @@ submodules:
 	# -rm -rf install/submodules/* && git submodule update --remote
 
 image:
-	@mkdir -p bin
-	@GOOS=linux go build -o bin/ trpc/trpc.go
-	@docker build -t trpc-cmdline:${VERSION} .
+	@docker build \
+	--build-arg GOPROXY="https://goproxy.cn" \
```

Comment:
Should we set this as the default? Alternatively, should we offer it as a separate build target or as an optional feature?

Reply:
modify GOPROXY,GOSUMDB to be optional